src/components/menu/_menu.js

```diff
@@ -0,0 +1,111 @@
+/**
```

Comment:
ensure that it gets concatenated first by the build, important because it defines the module